packages/components/carousel/src/Carousel.tsx

```diff
@@ -21,6 +23,8 @@ const Carousel: React.FC<CarouselProps> = ({
   interval = 3000,
   ...props
 }: CarouselProps) => {
+  const carouselRef = useRef<ResponsiveCarousel>(null);
+
   const renderContent = (content: Content): React.ReactNode => {
     switch (content.type) {
       case 'image':
```

Comment:
In the switch case to images, you can use our Picture component. For videos, you should do the same. Be sure to pull the latest components from the main branch to integrate them seamlessly with the entire DS.